### PR TITLE
Rolemaster Unified Official: Creature updates and misc QoL improvements.

### PR DESCRIPTION
**Pull request by nashidau**
- Show BMR info on front page
- Make BMR info more legible
- Don't add racial bonnus to BMR twice.
- Eyes/Skin/Hair now save
- Improve layout of the derived properties on the front page.
- Integrate EP into layout.
- Updates to way sheets open to make creatures easier to deal with.
- Force all sheets to be playersheets unless they are creatures
- Spell Lists -> click on the i to see the list in the compendium
- Add critical damage to the damage applied roll
- Add local copy of PragRoman
- Add select critical option
- Foe box looks a little pretier.
- Endurance:
  - Create correctly if zero
- Misc:
  - Handle 'Null' data
- Creatures;
  - They have critreduction, not critreduciton
  - Redo layout.
  - Can now target PCs
  - Track AP box
  - Status box (hits etc)
  - Creatures can attack!
  - Don't apply DB size mod twice.

# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

- [x] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.
- [ ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <   >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

### CODE REVIEW

File: RolemasterUnified_Official/rolemasterunified.html [hunk 1]
```diff
@@ -2300,10 +2414,13 @@ <h2>Statistics</h2>
...
   <h2>Actions</h2>
 
+ 
+
+
 
 
   
-  <div class='actionblock'>
+  <div class='actionblock' style='position:relatuve'>
```
Comment (kfroll20) [inline RolemasterUnified_Official/rolemasterunified.html:2423]: I wanted to point out the misspelling here for your inline styling

**Comment by kfroll20** [comment]
@nashidau, 

Thank you for all the work you've done to improve the sheet. There was just one typo for inline styling and I left a comment for it. Once that's addressed, I can merge this!

**Comment by nashidau** [comment]
Kayla; please tell me you used a tool to find that and were not reading pages of autogenerated cruft ;-)

Thank you; I've updated it.   I wonder if that was causing an issue I was seeing!?

**Comment by kfroll20** [comment]
> Kayla; please tell me you used a tool to find that and were not reading pages of autogenerated cruft ;-)
> 
> Thank you; I've updated it. I wonder if that was causing an issue I was seeing!?

Haha, I just happened to see it. We don't specifically look for things like that, but if we do come across them, we will point them out :)

**Pull request merged**